ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -628,7 +628,8 @@ public sealed class ArchiWebHandler : IDisposable {
 			{ "partner", steamID.ToString(CultureInfo.InvariantCulture) },
 			{ "serverid", "1" },
 			{ "trade_offer_create_params", !string.IsNullOrEmpty(token) ? new JsonObject { { "trade_offer_access_token", token } }.ToJsonText() : "" },
-			{ "tradeoffermessage", $"Sent by {SharedInfo.PublicIdentifier}/{SharedInfo.Version}" }
+			// Message length is limited to 128 characters.
+			{ "tradeoffermessage", !string.IsNullOrEmpty(message) ? (message.Length > 128 ? string.Concat(message.AsSpan(0, 125), "...") : message) : $"Sent by {SharedInfo.PublicIdentifier}/{SharedInfo.Version}" }
```

Comment:
Aside all the other stuff: `128` should be a `const` and the substring length should be derived from it. Furthermore, there is a continuation character containing 3 dots (reducing the amount of chars we use). Please take a look at [SteamChatMessage.cs](https://github.com/JustArchiNET/ArchiSteamFarm/blob/30c84efb5733bf2689a7a0956ec81dd38c01e20a/ArchiSteamFarm/Steam/Integration/SteamChatMessage.cs)